src/routes/Products.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { useFetchProducts } from "../store/products/products.hooks";
```

Comment:
I believe this to be a custom hook of your devising, whereas useEffect is a standard React hook.